test/models/deploys_test.rb

```diff
@@ -509,7 +509,7 @@ def generate_commits(amount:, stack_id:, user_id:, validate:)
       deploy1.save
 
       deploy2 = create_test_deploy(stack_id: stack_id, user_id: user_id, since_commit_id: commit_ids[1], until_commit_id: commit_ids[2])
-      deploy2.type = "Shipit::Fake"
+      deploy2.type = "Shipit::Task"
```

Comment:
I change this because the reload caused [`ActiveRecord::SubclassNotFound`](https://github.com/Shopify/shipit-engine/runs/615261539#step:6:240).